docs/UserGuide.adoc

```diff
@@ -545,6 +545,23 @@ Example:
 * `done icP/S7896432A icD/S4578902G res/fail`
 // end::mark[]
 
+// tag::delete[]
+=== Deleting a person : `delete`
+
+If you want to delete a person in ORGANice you can type in a command in the following format: `delete **NRIC**`  +
+ORGANice will prevent you to delete a person if the person you want to delete has the following criteria:
+
+* It is a patient and his status is currently under processing.
+* It is a donor and his status is currently under processing.
+* It is a doctor and he is in charge of some number of patients in ORGANice.
```

Comment:
They are a ... and their status ...

In this case "they" is singular and gender neutral